website/build.xml

```diff
@@ -25,13 +29,19 @@
     <mkdir dir="${parentdist.dir}"/>
   </target>
 
-  <target name="finished_tutorial.xml" depends="init"
+  <target name="compile-utilities" depends="init" description="Compile XOM utilities">
+    <javac srcdir="." destdir="." includeantruntime="false">
+      <include name="XOMTransformer.java"/>
+      <include name="XOMXIncluder.java"/>
+      <classpath refid="xom.class.path"/>
+    </javac>
+  </target>
+
+  <target name="finished_tutorial.xml" depends="compile-utilities"
           description="Merge original chapters and examples into a single file">
-    <java classname="nu.xom.samples.XIncludeDriver" output="${build.dir}/finished_tutorial.xml" fork="yes" failonerror="true">
+    <java classname="XOMXIncluder" output="${build.dir}/finished_tutorial.xml" fork="yes" failonerror="true">
```

Comment:
You don't need to change this. The existing code resolves thje XIncludes. This target should not change at all.